PhysicsTools/PatAlgos/plugins/PATIsolatedTrackProducer.cc

```diff
@@ -592,6 +665,25 @@ void pat::PATIsolatedTrackProducer::getIsolation(const PolarLorentzVector& p4,
   miniiso = pat::PFIsolation(chmiso, nhmiso, phmiso, pumiso);
 }
 
+float pat::PATIsolatedTrackProducer::getTrackIsolation(const PolarLorentzVector& track_p4,
+                                                       const math::XYZPoint& track_vtx,
+                                                       const float& track_dzError,
+                                                       const reco::TrackCollection* tracks) const {
+  float sumPt = 0.;
+  for (const auto& t : *tracks) {
+    // exclude tracks from different vertices by dz signficance
+    // similar to primary vertex assignment, but includes dzError from both tracks
+    if (fabs(t.dz(track_vtx)) >= trackIsolation_maxDZSig_ * hypot(t.dzError(), track_dzError))
```

Comment:
`std::abs` should be preferred to `fabs`
Could you please take the opportunity to replace also the other (two) `fabs` in this class?

Reply:
changes made